### DISCUSSION TITLE
Page content not updated

### DISCUSSION THREAD

**Discussion by xelber** [livewire/livewire#3352] [7. Bugs (archived)]
I have a simple component that looks like this
`
class ListQuote extends Component
{
    public $quotes;
    public string $query = '';
    public string $quoteStatus = '';

    public function render()
    {
        $quotes = Quote::where('user_id', Auth::id());
        if ($this->quoteStatus == 'active') {
            $quotes->where('status', 'CREATED');
        }

        $this->quotes = $quotes->get();
        return view('livewire.quote.list-quote');

    }
}
`

They are rendered using the following blade
![Screen Shot 2021-07-17 at 4 00 02 pm](https://user-images.githubusercontent.com/1886444/126027474-9138a670-ebf3-4236-a045-78c9f10996d5.png)

When the status drop-down changes, $quotes is updated but is not reflected on the page?? I can see the ajax is actually pulling the updated list.

**Comment by xelber** [comment]
It was to do with not having a root `<div>` - sorted. Thanks.

**Comment by ph7jack** [comment]
In livewire components and blade components must have one root element.
Various languages and frameworks are using this way, vuejs, react, laravel blade...
